src/back/hlsl/writer.rs

```diff
@@ -1176,10 +1176,7 @@ impl<'a, W: fmt::Write> super::Writer<'a, W> {
                         if !func.arguments.is_empty() {
                             write!(self.out, ", ")?;
                         }
-                        write!(
-                            self.out,
-                            "uint3 __global_invocation_id : SV_DispatchThreadID"
-                        )?;
+                        write!(self.out, "uint3 __global_invocation_id : SV_GroupThreadID")?;
```

Comment:
We probably want to change the HLSL variable name as well. It is no longer a "global invocation id".

Pre-existing, but: What happens if the entry point already has an argument that is the `BuiltIn::LocalInvocationId`? Can HLSL handle having two `SV_GroupThreadId` arguments on the same function?

Reply:
> Pre-existing, but: What happens if the entry point already has an argument that is the `BuiltIn::LocalInvocationId`? Can HLSL handle having two `SV_GroupThreadId` arguments on the same function?

Yeah, only msl and spv seem to complain.